src/helix/endpoints/moderation/ban_user.rs

```diff
@@ -80,10 +84,12 @@ pub struct BanUserBody {
     /// To put a user in a timeout, include this field and specify the timeout period, in seconds.
     /// The minimum timeout is 1 second and the maximum is 1,209,600 seconds (2 weeks).
     /// To end a user’s timeout early, set this field to 1, or send an Unban user request.
+    #[builder(default, setter(into))]
     pub duration: Option<u32>,
     /// The reason the user is being banned or put in a timeout. The text is user defined and limited to a maximum of 500 characters.
     pub reason: String,
```

Comment:
Not sure if this should have an `into`, since you have create/pass an owned `String` (compared to a `&str`).

Reply:
I think it's fine as is or with a `#[builder(setter(into))]`, definitely want to get this to not be owned though, but that's a much larger task as you already know.

Reply:
Sure, I added it.

> definitely want to get this to not be owned though, but that's a much larger task as you already know.

Yeah, at least having the requests take in references would be a huge step as it would reduce my use of `.clone()` by a lot.